example/.dart_tool/package_config.json

```diff
@@ -2,181 +2,145 @@
   "configVersion": 2,
   "packages": [
     {
-      "name": "archive",
```

Comment:
I think this file should be excluded from git.
https://github.com/flutter/flutter/blob/master/.gitignore
https://gist.github.com/Cheesetouched/ace446b9e6bedef76eb4fb7afb1ea164